rules/prefer-math-trunc.js

```diff
@@ -41,26 +30,41 @@ const bitwiseNotUnaryExpressionSelector = [
 
 const create = context => {
 	const sourceCode = context.getSourceCode();
+
 	const mathTruncFunctionCall = node => {
 		const text = sourceCode.getText(node);
 		const parenthesized = node.type === 'SequenceExpression' ? `(${text})` : text;
 		return `Math.trunc(${parenthesized})`;
 	};
 
 	return {
-		[bitwiseOrBinaryExpressionSelector]: node => {
-			context.report({
-				node,
-				messageId: MESSAGE_ID_BITWISE_OR,
-				fix: fixer => fixer.replaceText(node, mathTruncFunctionCall(node.left))
-			});
+		'BinaryExpression[right.type="Literal"]': node => {
+			const {operator, right} = node;
+			if (binaryOperators.has(operator) && right.value === 0) {
+				context.report({
+					node,
+					messageId: MESSAGE_ID_BITWISE,
+					data: {
+						operator,
+						value: right.raw
+					},
+					fix: fixer => fixer.replaceText(node, mathTruncFunctionCall(node.left))
+				});
+			}
 		},
-		[bitwiseOrAssignmentExpressionSelector]: node => {
-			context.report({
-				node,
-				messageId: MESSAGE_ID_BITWISE_OR,
-				fix: fixer => fixer.replaceText(node, `${sourceCode.getText(node.left)} = ${mathTruncFunctionCall(node.left)}`)
-			});
+		'AssignmentExpression[right.type="Literal"]': node => {
+			const {operator, right, left} = node;
+			if (assignmentOperators.has(operator) && right.value === 0) {
```

Comment:
```suggestion
			if (binaryOperators.has(operator.slice(0, -1)) && right.value === 0) {
```

So we don't have to add an operator in two places.